packages/block-library/src/button/edit.js

```diff
@@ -296,7 +298,7 @@ function ButtonEdit( props ) {
 						onClick={ startEditing }
 					/>
 				) }
-				{ isURLSet && isLinkTag && (
+				{ isURLSet && isLinkTag && ! metadata?.bindings?.url && (
```

Comment:
What if someone would like to unlink the button? In general, should it be possible to remove the binding in the UI?

Reply:
> In general, should it be possible to remove the binding in the UI?

I wasn't sure about that one. My reasoning was that, as right now bindings can only be added through the code editor, it shouldn't be a blocker if they can only be removed through the code editor as well. When we improve the UI and add the possibility to create bindings, we should provide mechanisms to remove them as well.

Anyway, I don't know which is the correct approach. Any thoughts?